profile/flatpak.sh.in

```diff
@@ -1,7 +1,13 @@
-# @sysconfdir@/profile.d/flatpak.sh - set XDG_DATA_DIRS
+# @sysconfdir@/profile.d/flatpak.sh - set XDG_DATA_DIRS and PATH
 
 if [ "${XDG_DATA_DIRS#*flatpak}" = "${XDG_DATA_DIRS}" ]; then
     XDG_DATA_DIRS="${XDG_DATA_HOME:-"$HOME/.local/share"}/flatpak/exports/share:@localstatedir@/lib/flatpak/exports/share:@externalinstalldir@/exports/share/:${XDG_DATA_DIRS:-/usr/local/share:/usr/share}"
 fi
 
 export XDG_DATA_DIRS
+
+if [ "${PATH#*flatpak}" = "${PATH}" ]; then
+    PATH="${XDG_DATA_HOME:-"$HOME/.local/share"}/flatpak/exports/bin:@localstatedir@/lib/flatpak/exports/bin:@externalinstalldir@/exports/bin:${PATH:-/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games}"
```

Comment:
Where's the default list taken from? And since our systems have the `/usr` merge, is there any need to have both `/usr/bin` and `/bin`? (Harmless, I know.)

Reply:
I copied from /etc/environment, which I believe is created by the libpam-modules postinst, of all things... But you're right. We can elide one. I also thought perhaps better to have the flatpak paths after the real ones...

Reply:
If it matches /etc/environment that's fine by me.